ast/ddl.go

```diff
@@ -1942,7 +1943,11 @@ func (n *PlacementOption) Restore(ctx *format.RestoreCtx) error {
 		case PlacementOptionPolicy:
 			ctx.WriteKeyWord("PLACEMENT POLICY ")
 			ctx.WritePlain("= ")
-			ctx.WriteName(n.StrValue)
+			if strings.ToLower(n.StrValue) == "default" {
+				ctx.WriteKeyWord("DEFAULT")
+			} else {
+				ctx.WriteName(n.StrValue)
+			}
```

Comment:
if you want `'DEFAULT'` and `DEFAULT` to be distinguishable you should create a separate `PlacementOptionType`. playing tricks in Restore() won't do much.

Reply:
yes. `"DEFAULT"` in parser.y always refer to the keyword. since `DEFAULT` is a reserved keyword it shouldn't collide with `Identifier`.

Reply:
Sounds great to me, I will try it.😆